internal/manifest/os.go

```diff
@@ -151,6 +151,10 @@ func (p *OS) getPackageSetChain() []rpmmd.PackageSet {
 		packages = append(packages, "chrony")
 	}
 
+	if p.OpenSCAPConfig != nil {
+		packages = append(packages, "openscap-scanner", "scap-security-guide")
```

Comment:
Do these have to be installed on the image, is it not sufficient to have them in the build root?

Reply:
So the `openscap` team said that customers would want to run scans again on the image after it has been built.
But to answer the question, no it's not necessary and we could add them in the build root.

Reply:
I think we should make it optional in the image, and let people install it if they want to. I think, but might be wrong, that you can run scans through satellite and insights without anything installed. Might be worth checking.